### PR TITLE
fix: add invite refresh to the listener

### PR DESCRIPTION
**Pull request by ricantar**

### CODE REVIEW

File: cache/cache.go [hunk 1]
```diff
@@ -992,6 +992,8 @@ func (c *Cache) updateUserAccess(inUser types.UserAccess) {
 		if err != nil {
 			fmt.Printf("error upserting user permissions: %v\n", err)
 		}
+
+		c.pendingLoadBalancersMapByEmail[inUser.Email] = removeLoadBalancerByID(c.pendingLoadBalancersMapByEmail[inUser.Email], lbID)
```
Comment (commoddity) [inline cache/cache.go:996]: This should be in an if block eg. `if inUser.Accepted { ... }`

Comment (ricantar) [reply cache/cache.go:996]: It's

Comment (commoddity) [reply cache/cache.go:996]: Ah sorry didn't  see that above, my bad. LGTM

**Review by commoddity — APPROVED** [review]
LGTM

**Pull request merged**